### DISCUSSION TITLE
Storybook relating that repository is empty when I have one story.

### DISCUSSION THREAD

**Discussion by joao-pedrozo** [storybookjs/storybook#14066] [Help]
**Describe the bug**
I have a vue.js application which I want to install storybook. I run `npx sb init` to start, then I add one story in the related path informed on `.storybook/main.js`, but it says that my Storybook is empty! Another curious thing: it says that my Storybook is empty, but if I change a bit of the only story it has, the story **appears but the component is completly raw and without styles**, and if I refresh the page, this stories that shows up when I saved the file disappears again.

**To Reproduce**

1. Clone the repo(https://github.com/ConsultaRemedios/cr-ui)
2. Run `yarn`
3. Run `npx sb init`
4. Change the "stories" on `.storybook/main.js` to `"../src/components/**/**/*.stories.js"`
5. Run yarn storybook

**Expected behavior**
The component related on the story should appear correctly.

**Screenshots**
![image](https://user-images.githubusercontent.com/40037644/109215192-947b3d80-7791-11eb-8089-e0e056c2577e.png)

**System**
```
Environment Info:

  System:
    OS: Linux 5.4 Ubuntu 20.04.2 LTS (Focal Fossa)        
    CPU: (12) x64 AMD Ryzen 5 1600 Six-Core Processor     
  Binaries:
    Node: 15.10.0 - ~/.nvm/versions/node/v15.10.0/bin/node
    Yarn: 1.22.10 - ~/.nvm/versions/node/v15.10.0/bin/yarn
    npm: 7.5.3 - ~/.nvm/versions/node/v15.10.0/bin/npm    
  npmPackages:
    @storybook/addon-actions: ^6.1.20 => 6.1.20
    @storybook/addon-essentials: ^6.1.20 => 6.1.20        
    @storybook/addon-links: ^6.1.20 => 6.1.20
    @storybook/vue: ^6.1.20 => 6.1.20
```

**Comment by shilman** [comment]
I don't see any stories under your `src/components` directory
